lib/ae_mdw/aex9.ex

```diff
@@ -85,6 +105,159 @@ defmodule AeMdw.Aex9 do
     )
   end
 
+  @spec fetch_tokens(pagination(), query(), order_by(), cursor() | nil) ::
+          {:ok, cursor() | nil, [aex9_token()], cursor() | nil} | {:error, Error.t()}
+  def fetch_tokens(pagination, query, order_by, cursor) do
+    try do
```

Comment:
fallback_controller would be helpful to standardize wherever an exception happens it's raised immediately in place of being returned as a struct.

Reply:
okay the other way around like this.